deployment/k8s/README.md

```diff
@@ -0,0 +1,56 @@
+# Deploy Service Manager on Kubernetes on Minikube:
+
+## Build a docker image
+
+You can set the docker client to the Minikube docker environment by executing
+* windows: ```@FOR /f "tokens=*" %i IN ('minikube docker-env') DO @%i```
+* linux/mac: ```eval $(minikube docker-env)```
+
+and build the image
+
+```sh
+docker build -t "service-manager:latest" -f Dockerfile .
+```
+
+Alternatively you can build a docker image and push it to an external repository.
+In this case you have to specify the image, tag and pullPolicy in the Service Manager helm install command.
+
+```sh
+docker build -t "<image_name>:<tag>" -f Dockerfile .
+docker push "<image_name>:<tag>"
+```
+
+## Install Service Manager
+
+Go to *deployment/k8s/charts/service-manager* folder.
+
+Execute:
+
+```sh
+helm dependency update
+```
+
+to get the required dependencies.
+
+To install the Service Manager and PostgreSQL database, execute:
+
+```sh
+helm install --name service-manager --namespace service-manager .
+```
+
+Alternatively you can use different PostgreSQL username or password:
+
+```sh
+helm install --name service-manager --namespace service-manager . --set postgresql.postgresUser=<pguser> --set postgresql.postgresPassword=<pgpass>
+```
+
+Or install only Service Manager with external PostgreSQL:
+
+```sh
+helm install --name service-manager --namespace service-manager . --set postgresql.enabled=false --set postgresql.uri=<postgresql_connection_string>
```

Comment:
Unfortunately it is not possible to remove `postgresql.enabled` (it acts as a toggle for postgresql helm chart installation). Maybe it could be renamed to `postgresql.install` and `postgresql.uri` could be `externalPostgresURI`. This way there will be better separation between the use cases. It will become `postgresql.install=false` and `externalPostgresURI =<uri>` or `postgresql.install=true`.